addons/ballistics/CfgWeapons.hpp

```diff
@@ -11,19 +11,18 @@ class CfgWeapons {
         ACE_barrelLength = 730;
         ACE_barrelTwist = 381.0;
         initSpeed = -1.0;
-        magazineWell[] += {
-            "CBA_50BMG_M107"
```

Comment:
why remove a valid magazine well? This one should be added on CBA side I'd say

Reply:
The both work in-game. As you want.
Maybe the name of this CBA magwell should be changed,
it's not really appropriate to the vanilla game and this rifle.